### PR TITLE
Feat/upgrade wasmbus rpc

### PR DESCRIPTION
**Pull request by stevelr**
update to wasmbus-rpc 0.8.2

### CODE REVIEW

File: factorial/rust/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-interface-factorial"
-version = "0.4.0"
+version = "0.4.1"
```
Comment (brooksmtownsend) [inline factorial/rust/Cargo.toml:3]: I think this should be a minor version bump for all interfaces, the bump of `wasmbus-rpc` `0.7` -> `0.8.2` will cause actors that use this interface and an older version of wasmbus-rpc to fail to compile

File: blobstore/rust/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-interface-blobstore"
-version = "0.1.1"
+version = "0.1.2"
```
Comment (brooksmtownsend) [inline blobstore/rust/Cargo.toml:3]: This should be a minor bump to `0.2.0`

Comment (brooksmtownsend) [reply blobstore/rust/Cargo.toml:3]: Unless `0.1.0` was published with wasmbus-rpc `0.8`, in which case disregard

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]

File: factorial/rust/Cargo.toml (hunk 1 above)
Comment (stevelr) [reply factorial/rust/Cargo.toml:3]: I think it needs to be semver minor bump - too many cross-dependencies. An actor will only have an issue if they run `cargo update`, but if they do then we should have updated wasmbus-rpc and the applicable interfaces by then.

File: blobstore/rust/Cargo.toml (hunk 2 above)
Comment (stevelr) [reply blobstore/rust/Cargo.toml]: done

File: factorial/rust/Cargo.toml (hunk 1 above)
Comment (stevelr) [reply factorial/rust/Cargo.toml:3]: sorry - misread your response - yes all interfaces get a semver minor bump. The last published crate for this interface was 0.3.x.

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**